src/apis/ActionSheetIOS.re

```diff
@@ -10,8 +10,7 @@ external options:
     ~tintColor: Color.t=?,
     unit
   ) =>
-  options =
-  "";
```

Comment:
I don't know why this is being removed. It this the formatter?

Reply:
With `reason` version 3.6, that `= ""` bit has become optional. See the relevant [PR](https://github.com/facebook/reason/pull/2464) and [issue](https://github.com/facebook/reason/issues/2422) for more information.

As you correctly guessed, those are removed when `refmt` is run.